lib/ansible/modules/cloud/azure/azure_rm_networkinterface.py

```diff
@@ -349,6 +374,7 @@ def nic_to_dict(nic):
     private_ip_allocation_method=dict(type='str', choices=['Dynamic', 'Static'], default='Dynamic'),
     public_ip_address_name=dict(type='str', aliases=['public_ip_address', 'public_ip_name']),
     public_ip_allocation_method=dict(type='str', choices=['Dynamic', 'Static'], default='Dynamic'),
+    load_balancer_backend_address_pools=dict(type='list'),
```

Comment:
just curious, isn't this a config in load balancer, why add in azure_network_interface module?

Reply:
in `loadbalancer` SDK, it simply ignore this param. And this API is exposed in NIC.

Reply:
I suppose that before the receiving NIC has been created it doesn't make sense to add it to a load balancer. Reduces risk of faulty config atleast.